custom/icds_reports/utils.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, domain, filters, report_id):
         self.report_config = report_config
 
     def data(self):
-        return self.report_config.get_data()
+        return self.report_config.data_source._get_data().values()
```

Comment:
@emord because in ```_get_data()``` we doesn't use formatter which works very slowly.

For example: locally I generated 100000 records in one datasource.
when we use ```get_data()``` - we get timeout.
when we use ```_get_data()``` - report load in 1 min